src/components/Advertisements/core/AdvertisementEntry/AdvertisementEntry.tsx

```
@@ -133,53 +127,123 @@ function AdvertisementEntry({
+                <div className={styles.mediaContainer}>
+                  {advertisement.attachments &&
+                  advertisement.attachments.length > 0 ? (
+                    advertisement.attachments.length > 1 ? (
+                      <Carousel className={styles.carouselContainer}>
+                        {advertisement.attachments.map((attachment, index) => (
+                          <Carousel.Item key={index}>
+                            <div className={styles.imageWrapper}>
+                              <img
+                                className={`d-block w-100 ${styles.cardImage}`}
+                                src={attachment.url}
+                                alt={`Advertisement image #${index + 1} for ${advertisement.name ?? 'ad'}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)